internal/providers/node_runtime_deps_info.bzl

```diff
@@ -70,6 +70,10 @@ def run_node(ctx, inputs, arguments, executable, chdir = None, **kwargs):
         arguments: list or ctx.actions.Args object containing arguments to pass to the executable
         executable: stringy representation of the executable this action will run, eg eg. "my_executable" rather than ctx.executable.my_executable
         chdir: directory we should change to be the working dir
+        stamp_env_vars: if True ctx.version_file and ctx.info_file are supplied as inputs to the action
```

Comment:
oh this is just `stamp` which is a boolean - but users of npm_package_bin (I guess pkg_npm in your case) need a trinary field where they can say stamp=always or stamp=never or stamp=use_stamp_flag, that's what gets plumbed to here

Reply:
Refactored so run_node sniffs out the "stamp" var which is tri-state